### PR TITLE
docs!: fix wrong info at variable outputs page

### PR DESCRIPTION
**Pull request by Torres-ssf**
This is a simple fix for the wrong information I've added when writing the doc page `variable-outputs`.

I've also:

- Rename `getAssetId` to `getMintedAssetId`.
- Move the code snippet related to the `variable-outputs` page to its test suite file.

## Breaking Changes
- [Rename](https://github.com/FuelLabs/fuels-ts/pull/1962/commits/022331ba03af325ea1668af66c1eaafec5b53513) `getAssetId` to `getMintedAssetId`

### CODE REVIEW

File: apps/docs-snippets/src/guide/contracts/variable-outputs.test.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+/**
+ * @group node
+ */
+describe(__filename, () => {
+  let contract: Contract;
+  beforeAll(async () => {
+    contract = await createAndDeployContractFromProject(DocSnippetProjectsEnum.TOKEN);
+  });
+
+  it('should successfully execute contract call with txParams', async () => {
```
Comment (danielbate) [inline apps/docs-snippets/src/guide/contracts/variable-outputs.test.ts:16]:
```suggestion
  it('should successfully execute contract call with variable outputs', async () => {
```

Just making it more explicit as we have lots of tx param tests and #1961 introduces many

File: apps/docs/src/guide/contracts/variable-outputs.md [hunk 2]
```diff
@@ -6,22 +6,20 @@ When using these transfer functions within your Sway projects, it is important t
 
 For instance, if a contract function calls a Sway transfer function 3 times, it will require 3 Output Variables present within the list of outputs in your transaction.
 
-## Example: Sway's built-in functions that requires `Output Variable`
+## Example: Sway's functions that requires `Output Variable`
```
Comment (danielbate) [inline apps/docs/src/guide/contracts/variable-outputs.md:9]:
```suggestion
## Example: Sway functions that requires `Output Variable`
```

File: .changeset/empty-buses-film.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/transactions": patch
```
Comment (petertonysmith94) [inline .changeset/empty-buses-film.md:2]:
```suggestion
"@fuel-ts/transactions": minor
```

Is this a minor change?

**Comment by fuel-service-user** [comment]
✨ A PR has been created under the `rc-1962` tag on `fuels-wallet` repo.
https://github.com/FuelLabs/fuels-wallet/pull/1208

**Review by danielbate — APPROVED** [review]

**Review by petertonysmith94 — APPROVED** [review]

**Pull request merged**